### PR TITLE
Php doc improvements

### PR DESCRIPTION
**Pull request by Samistine**
# Summary

# Checklist

- [x] Added changelog entry
- [X] Ran unit tests (Check the README for instructions)

### CODE REVIEW

File: lib/Braintree/Version.php [hunk 1]
```diff
@@ -12,7 +12,7 @@ class Version
      */
     const MAJOR = 3;
     const MINOR = 25;
-    const TINY = 0;
+    const TINY = 1;
```
Comment (bluk) [inline lib/Braintree/Version.php]: The release process takes care of upgrading the constant, so can this change be reverted?

File: lib/Braintree/Transaction.php [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <?php
 namespace Braintree;
+use Braintree\Transaction\AndroidPayCardDetails;
```
Comment (bluk) [inline lib/Braintree/Transaction.php]: This seems unnecessary.

File: lib/Braintree/Transaction.php [hunk 3]
```diff
@@ -147,30 +148,30 @@
  * @property-read string $cvvResponseCode
  * @property-read string $id transaction id
  * @property-read string $amount transaction amount
- * @property-read Braintree\Transaction\AddressDetails $billingDetails transaction billing address
+ * @property-read \Braintree_Transaction_AddressDetails $billingDetails transaction billing address
```
Comment (bluk) [inline lib/Braintree/Transaction.php]: Most of the phpdoc in the other files use a namespaced variation for property types. Does this change help in a particular IDE or development environment for you?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,9 @@
+## 3.25.1
```
Comment (crookedneighbor) [inline CHANGELOG.md]: Can you just mark this as `# unreleased`? We don't yet know if the next release will be a point release or a minor release.

File: lib/Braintree/Version.php (hunk 1 above)
Comment (crookedneighbor) [reply lib/Braintree/Version.php]: Agreed. This change should be removed.

**Comment by crookedneighbor** [comment]
@Samistine are you going to make the requested updates?

**Comment by crookedneighbor** [comment]
Going to close this for inactivity. Feel free to comment here if you'd like me to re-open it.

**Pull request closed without merging**